### PR TITLE
Unsubscribe from emails

### LINKED ISSUE #62: Images are missing during local development

### LINKED ISSUE #63: Unsubscribe from future communication
What does this feature look like?

### LINKED ISSUE #68: Mark email addresses as confirmed
The same table could also store unsubscribed emails for #63.

### PR DESCRIPTION
**Pull request by jacebrowning**
Closes #62 
Closes #63 
Closes #68

### CODE REVIEW

File: api/core/helpers.py [hunk 1]
```diff
@@ -46,5 +40,22 @@ def send_login_email(user, request, *, welcome):
...
+    return base + token
+
+
+def get_site_url(request):
+    return reverse('index', request=request)
+
+
+def get_unsubscribe_url(request, user):
+    # TODO: Change "login" to a new frontend path
+    base = reverse('redirector', args=["login"], request=request)
```
Comment (jacebrowning) [inline api/core/helpers.py]: We can change "login" to "unsubscribed" or whatever path gets implemented on the frontend.

Comment (diiq) [reply api/core/helpers.py]: State added to frontend.

File: Makefile [hunk 2]
```diff
@@ -122,8 +122,7 @@ uml: install
 
 .PHONY: run
 run: install ## Run the applicaiton
-	mkdir -p web_client/build
-	cp web_client/app/index.html web_client/build/index.html
+	yarn build
```
[the remark this answers is not in the document]
Comment (diiq) [reply Makefile]: I seem to be having some trouble getting live updates on the frontend in dev, now? Might be unrelated, haven't investigated yet.

**Comment by jacebrowning** [comment]
I can confirm that something is up with hot reloading (the page refreshes, but not the content), but I also don't see any changes on this PR that would affect this, so I guess it's been broken for a while. 😞

**Pull request merged**